components/DesktopNav.tsx

```diff
@@ -0,0 +1,232 @@
+import { useTranslation } from "next-i18next"
+import React from "react"
+import Image from "react-bootstrap/Image"
+import styled from "styled-components"
+import { SignInWithButton, signOutAndRedirectToHome, useAuth } from "./auth"
+import { Container, Dropdown, Nav, NavDropdown } from "./bootstrap"
+import { useProfile } from "./db"
+import { NavLink } from "./Navlink"
+
+export const DesktopNav: React.FC<React.PropsWithChildren<unknown>> = () => {
+  const { authenticated, user } = useAuth()
+  const { t } = useTranslation(["common", "auth"])
+  const userLink = "/profile?id=" + user?.uid
+
+  const result = useProfile()
+  let isOrg = result?.profile?.role === "organization"
+
+  const LogoBox = styled.div`
```

Comment:
Is it necessary to manually set the widths for these? I feel like a fixed font-size  + padding/spacing should keep the sizing consistent anyway.

Reply:
I'm not sure why I did that; I must have chased down some rabbit hole.

These are not necessary and have been removed.